src/components/ol/Map.vue

```diff
@@ -266,6 +266,8 @@ export default {
       // wrap the tooltip span in a OL overlay and add it to map
       me.overlay = new Overlay({
         element: overlayEl,
+        stopEvent: false,
+        className: 'wgu-hover-ol-overlay',
```

Comment:
why is `classname` necessary?
I cannot find a reference to it in the code base.

Reply:
See section `stopEvent` in https://openlayers.org/en/latest/apidoc/module-ol_Overlay-Overlay.html